Documentation/data-collection.md

```diff
@@ -238,6 +238,10 @@ data:
     - '{__name__="che_workspace_start_time_seconds_count"}'
     # (cloud credential operator, @openshift/openshift-team-hive) Track current mode the cloud-credentials-operator is functioning under.
     - '{__name__="cco_credentials_mode"}'
+    # (image-registry-team) :apiserver_v1_image_imports:sum counts the number of image stream
```

Comment:
Why is there no prefix on this metric?  Was there a reason for that?

Reply:
I can't actually understand why the metric name is bad (i saw lili's question above)

Reply:
@smarterclayton the name was suggested by @simonpasquier at https://github.com/openshift/cluster-image-registry-operator/pull/626#discussion_r503920668

Reply:
It looks like this is following a convention from upstream docs that we don't follow elsewhere, so we have one metric that is different from all the others.